src/Microsoft.AspNet.OData.Shared/IDeltaSet.cs

```diff
@@ -0,0 +1,20 @@
+﻿//-----------------------------------------------------------------------------
+// <copyright file="IDeltaSet.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// Basic interface to reperesent a Deltaset which is a collection of Deltas.
+    /// This is being implemented by DeltaSet<TStructuralType/>. Since it's being implemented by a generic type and
+    /// since we need to check in a few places (like deserializer) where the object is a DeltaSet and the {TStructuralType} is not available,
```

Comment:
```suggestion
    /// since we need to check in a few places (like deserializer) whether the object is a DeltaSet and the {TStructuralType} is not available,
```